apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -382,7 +382,7 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
                 }}
               />
               {selectedLocation && LocationOptions}
-              <DialogFooter className="mt-4">
+              <DialogFooter className="relative">
```

Comment:
@ujjwal2505 this seem to be working. can you pull the changes and test it once

Reply:
I tested the code and it is working. We can close the issue.